### PR TITLE
PROD-4997 Create documentation for v2 policies

### PR DESCRIPTION
**Pull request by Jerjomins**

### CODE REVIEW

File: source/rest_api/create_unpaid_v1_policy.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+~~~~~~~~~~
+.. code:: bash
+
+	POST https://api.kasko.io/policies
+
+
+Parameters
+~~~~~~~~~~
++----------------------------+------------+---------------+---------------------------------------------------------+
+| Parameter                  | required   | Type          | Description                                             |
```
Comment (tomstumshais) [inline source/rest_api/create_unpaid_v1_policy.rst]:
```suggestion
| Parameter                  | Required   | Type          | Description                                             |
```

File: source/rest_api/create_unpaid_v1_policy.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
++----------------------------+------------+---------------+---------------------------------------------------------+
+| last_name                  | yes        | string        |  Last Name of the customer                              |
++----------------------------+------------+---------------+---------------------------------------------------------+
+| email                      | yes        | string        |  Email address of the customer                          |
++----------------------------+------------+---------------+---------------------------------------------------------+
+| language                   | yes        | string        |  Two letter ISO 639‑1 language code                     |
++----------------------------+------------+---------------+---------------------------------------------------------+
+| distributor_traffic_source | no         | string        |  Distributor reference of traffic source.  e.g. Email   |
++----------------------------+------------+---------------+---------------------------------------------------------+
+| data                       | maybe      | JSON          |  Refer to the product section here for required data    |
```
Comment (tomstumshais) [inline source/rest_api/create_unpaid_v1_policy.rst]: I guess there should be only `yes` or `no`, not `maybe`.

File: source/rest_api/create_unpaid_v2_policy.rst [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+~~~~~~~~~~
+.. code:: bash
+
+	POST https://api.kasko.io/policies
+
+
+Parameters
+~~~~~~~~~~
++----------------------------+------------+---------------+---------------------------------------------------------+
+| Parameter                  | required   | Type          | Description                                             |
```
Comment (tomstumshais) [inline source/rest_api/create_unpaid_v2_policy.rst]:
```suggestion
| Parameter                  | Required   | Type          | Description                                             |
```

File: source/rest_api/create_unpaid_v2_policy.rst [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
++----------------------------+------------+---------------+---------------------------------------------------------+
+| first_name                 | yes        | string        |  First Name of the customer                             |
++----------------------------+------------+---------------+---------------------------------------------------------+
+| last_name                  | yes        | string        |  Last Name of the customer                              |
++----------------------------+------------+---------------+---------------------------------------------------------+
+| email                      | yes        | string        |  Email address of the customer                          |
++----------------------------+------------+---------------+---------------------------------------------------------+
+| language                   | yes        | string        |  Two letter ISO 639‑1 language code                     |
++----------------------------+------------+---------------+---------------------------------------------------------+
+| data                       | maybe      | JSON          |  Refer to the product section here for required data    |
```
Comment (tomstumshais) [inline source/rest_api/create_unpaid_v2_policy.rst]: Same with `maybe`

File: source/rest_api/create_unpaid_v1_policy.rst [hunk 5]
```diff
@@ -0,0 +1,64 @@
+.. _create_unpaid_v1_policy:
+
+Create an unpaid v1 policy
+---------------------------------
+Create an unpaid policy on the Kasko platform.
+
+Why do we do this 2 step mechanism?   We want to ensure that there is no problem with the quote or details before you charge your customers for the policy.  After this point we can ensure that the policy can be purchased.
```
Comment (MatissJanis) [inline source/rest_api/create_unpaid_v1_policy.rst]:
```suggestion
Why do we do this 2 step mechanism?   We need to ensure that there is no problem with quote or policy details before a charge can be made for the customer.  With this we ensure that a policy can be purchased.
```

File: source/rest_api/create_unpaid_v2_policy.rst [hunk 6]
```diff
@@ -0,0 +1,65 @@
+.. _create_unpaid_v2_policy:
+
+Create an unpaid v2 policy
+---------------------------------
+Create an unpaid policy on the Kasko platform.
+
+Why do we do this 2 step mechanism?   We want to ensure that there is no problem with the quote or details before you charge your customers for the policy.  After this point we can ensure that the policy can be purchased.
```
Comment (MatissJanis) [inline source/rest_api/create_unpaid_v2_policy.rst]:
```suggestion
Why do we do this 2 step mechanism?   We need to ensure that there is no problem with quote or policy details before a charge can be made for the customer.  With this we ensure that a policy can be purchased.
```

File: source/rest_api/create_unpaid_v2_policy.rst [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+
+Parameters
+~~~~~~~~~~
++----------------------------+------------+---------------+---------------------------------------------------------+
+| Parameter                  | Required   | Type          | Description                                             |
++============================+============+===============+=========================================================+
+| quote_token                | yes        | string        |  Quote token provided in quote object                   |
++----------------------------+------------+---------------+---------------------------------------------------------+
+| first_name                 | yes        | string        |  First Name of the customer                             |
```
Comment (MatissJanis) [inline source/rest_api/create_unpaid_v2_policy.rst]:
```suggestion
| first_name                 | yes        | string        |  First name of the customer                             |
```

File: source/rest_api/create_unpaid_v2_policy.rst [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
++----------------------------+------------+---------------+---------------------------------------------------------+
+| last_name                  | yes        | string        |  Last Name of the customer                              |
```
Comment (MatissJanis) [inline source/rest_api/create_unpaid_v2_policy.rst]:
```suggestion
| last_name                  | yes        | string        |  Last name of the customer                              |
```

File: source/rest_api/create_unpaid_v1_policy.rst [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+
+Parameters
+~~~~~~~~~~
++----------------------------+------------+---------------+---------------------------------------------------------+
+| Parameter                  | Required   | Type          | Description                                             |
++============================+============+===============+=========================================================+
+| quote_token                | yes        | string        |  Quote token provided in quote object                   |
++----------------------------+------------+---------------+---------------------------------------------------------+
+| first_name                 | yes        | string        |  First Name of the customer                             |
```
Comment (MatissJanis) [inline source/rest_api/create_unpaid_v1_policy.rst]:
```suggestion
| first_name                 | yes        | string        |  First name of the customer                             |
```

File: source/rest_api/create_unpaid_v1_policy.rst [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
++----------------------------+------------+---------------+---------------------------------------------------------+
+| last_name                  | yes        | string        |  Last Name of the customer                              |
```
Comment (MatissJanis) [inline source/rest_api/create_unpaid_v1_policy.rst]:
```suggestion
| last_name                  | yes        | string        |  Last name of the customer                              |
```

File: source/rest_api/create_unpaid_v2_policy.rst [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+
+
+Example Response
+~~~~~~~~~~~~~~~~
+
+.. code:: javascript
+
+	{
+	  "id": "lpol_zWx47B5qY6wXMLPNREA9dDnOQVZ3",
+      "insurer_policy_id":"XXXX-12345678",
```
Comment (MatissJanis) [inline source/rest_api/create_unpaid_v2_policy.rst]:
```suggestion
	  "insurer_policy_id":"XXXX-12345678",
```

File: source/rest_api/create_unpaid_v2_policy.rst [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+.. code:: bash
+
+	curl https://api.kasko.io/policies \
+	    -u sk_test_SECRET_KEY: \
+        -H 'Accept: application/vnd.kasko.v2+json' \
+	    -d quote_token=QUOTE_TOKEN \
+	    -d first_name=FIRSTNAME \
+	    -d last_name=SURNAME \
+	    -d email=EMAIL_ADDRESS \
+        -d language=LANGUAGE \
```
Comment (MatissJanis) [inline source/rest_api/create_unpaid_v2_policy.rst]:
```suggestion
	    -d language=LANGUAGE \
```

File: source/rest_api/create_unpaid_v2_policy.rst [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
++----------------------------+------------+---------------+---------------------------------------------------------+
+
+Example Request
+~~~~~~~~~~~~~~~
+
+.. code:: bash
+
+	curl https://api.kasko.io/policies \
+	    -u sk_test_SECRET_KEY: \
+        -H 'Accept: application/vnd.kasko.v2+json' \
```
Comment (MatissJanis) [inline source/rest_api/create_unpaid_v2_policy.rst]:
```suggestion
	    -H 'Accept: application/vnd.kasko.v2+json' \
```

File: source/rest_api/index.rst [hunk 14]
```diff
@@ -90,69 +90,22 @@ Depending if you are using an old product (pre 2018) or a new product (post 2018
...
-	    -d first_name=FIRSTNAME \
-	    -d last_name=SURNAME \
-	    -d email=EMAIL_ADDRESS \
-	    -d data=DATA
-
+Create unpaid policy
+--------------------
+Create policy on KASKO platform
 
+Depending if you are using v1 policy or v2 policy, there will be a different approach to this step
```
Comment (MatissJanis) [inline source/rest_api/index.rst]:
```suggestion
Depending wether you are using v1 policy or v2 policy, there will be a different approach to this step.
```

File: source/rest_api/index.rst [hunk 15]
```diff
@@ -90,69 +90,22 @@ Depending if you are using an old product (pre 2018) or a new product (post 2018
...
-	{
-	  "id": "tmGgyzWx47B5qY6wXMLPNREA9dDnOQVZ3",
-	  "payment_token": "2pwqBTy+79gK/dKuJmRjC1yTk7jx5zvuh5tn34139GiOd8irZuuTB6ViTKyRMNW8VcctGzDAn+QQf9fHOjdowpE67GHEFFuy4X+QFfx87qlg=",
-	  "_links": {
-	    "_self": {
-	      "href": "https://api.kasko.io/policies/tmGgyzWx47B5qY6wXMLPNREA9dDnOQVZ3"
-	    }
-	  }
-	}
+Depending if you are using v1 policy or v2 policy, there will be a different approach to this step
```
Comment (MatissJanis) [inline source/rest_api/index.rst]:
```suggestion
Depending wether you are using v1 policy or v2 policy, there will be a different approach to this step.
```

**Pull request merged**